### ISSUE TITLE
Does verbose work?

### ISSUE THREAD

**Issue by mwaskom** [nipy/PySurfer#160]
The `verbose` parameter, specifically in `project_volume_data`, doesn't seem to have any effect. e.g.:

```python
surf = project_volume_data(vol, hemi, reg,
                               smooth_fwhm=1, projarg=(.5, .5, 1),
                               verbose=verbose)
```

Prints (to `stderr`, based on the Jupyter css color) INFO level log information no matter whether I set `verbose` to `False` or `"DEBUG"`.

Do I not understand how to use it, or is this parameter being ignored?

**Comment by Eric89GXL** [comment]
That function is missing the `@verbose` decorator

**Comment by mwaskom** [comment]
Easy fix :) #161
